python/ray/tests/test_k8s_operator_examples.py

```diff
@@ -0,0 +1,150 @@
+"""Tests launch and teardown of multiple Ray clusters using Kubernetes
+operator."""
+import sys
+import os
+import subprocess
+import tempfile
+import time
+import unittest
+
+import kubernetes
+import pytest
+import yaml
+
+IMAGE_ENV = "KUBERNETES_OPERATOR_TEST_IMAGE"
+IMAGE = os.getenv(IMAGE_ENV, "rayproject/ray:nightly")
+NAMESPACE = "test-k8s-operator-examples"
+
+
+def retry_until_true(f):
+    # Retry 60 times with 1 second delay between attempts.
+    def f_with_retries(*args, **kwargs):
+        for _ in range(60):
+            if f(*args, **kwargs):
```

Comment:
Extended the timeout from 30 retries to 60 to get it to consistently pass on minikube.